server/src/main/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpoints.java

```diff
@@ -161,11 +184,14 @@ public ScimUserEndpoints(
         this.passwordValidator = passwordValidator;
         this.codeStore = codeStore;
         this.approvalStore = approvalStore;
+        this.aliasEntitiesEnabled = aliasEntitiesEnabled;
         this.userMaxCount = userMaxCount;
         this.membershipManager = membershipManager;
         this.messageConverters = new HttpMessageConverter[] {
                 new ExceptionReportHttpMessageConverter()
         };
+        this.aliasHandler = aliasHandler;
+        this.transactionTemplate = new TransactionTemplate(transactionManager);
```

Comment:
hard to test if you create the TransactionTemplate in code here , so not easy to mock then

Reply:
fixed with commit b94a96d

Reply:
ok